bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TTbar01Jets_TypeIHeavyN/TTbar01Jets_TypeIHeavyN-El_LepSMTop_3L_LO_MN40_XQCUT20/TTbar01Jets_TypeIHeavyN-El_LepSMTop_3L_LO_MN40_XQCUT20_customizecards.dat

```diff
@@ -0,0 +1,18 @@
+set param_card mn1 40
+set param_card vmun1 0.0
+set param_card ven1 0.01
+set param_card wn1 AUTO
+set param_card mn2 999999
+set param_card mn3 999999
+set param_card ven2 0.0
+set param_card ven3 0.0
+set param_card vmun2 0.0
+set param_card vmun3 0.0
+set param_card vtan1 0.0
+set param_card vtan2 0.0
+set param_card vtan3 0.0
+set param_card wn2 100
+set param_card wn3 100
+set param_card mass 6 172.9
```

Comment:
sorry. small remark. the background will have 172.5. wouldn't it be better to make a consistent choice of mtop. of course there will be no impact on the result. more a general remark also for the write-up and description of what was done.

Reply:
I just checked one other TTbar samples, and that didn't change the width for top. So I think giving mtop=yukawatop=172.5 without modifying the widths should be okay. Right?